src/PIL/_imaging.pyi

```diff
@@ -1,3 +1,18 @@
 from typing import Any
 
+from typing_extensions import Buffer
```

Comment:
Just pointing out that is adding typing_extensions as a requirement. Previous discussion about this can be seen in #7642

Reply:
This is not super critical. We could use `bytes | bytesarray | memoryview` here, instead. This would exclude other types using the buffer protocol, but I'm not sure how common that is with Pillow. (And it can always be `# type: ignore`d, of course.)

Reply:
I removed this in https://github.com/srittau/Pillow/pull/2